libbpfgo.go

```diff
@@ -533,7 +486,7 @@ func (m *Module) Close() {
 	}
 	for _, link := range m.links {
 		if link.link != nil {
-			C.bpf_link__destroy(link.link)
+			link.Destroy()
```

Comment:
Reminder(myself): to check tcProbes and tracingProbes Destroy() for errors once we bump tracee.